peps/pep-2026.rst

```diff
@@ -0,0 +1,570 @@
+(see `rejected ideas <PEP 2026 rejected_>`_): we keep 3 as the major version,
+and the minor version is still two digits.
+The minor will eventually change to three digits but this is predictable,
+a long way off and can be planned for.
+
+We retain the ``python3`` executable.
+
+Version mapping
+---------------
+
+Versions 3.15 to 3.25 inclusive will be skipped.
+Features, deprecations and removals planned for these will be remapped to the
+new version numbers.
+
+For example, a deprecation initially planned for removal in 3.16 will instead
+be removed in 3.27.
+
++-------------+------------------+-----------------+
+| Old version | New version      | Initial release |
++=============+==================+=================+
+| 3.14        | 3.14 (no change) | 2025            |
++-------------+------------------+-----------------+
+| 3.15        | 3.26             | 2026            |
++-------------+------------------+-----------------+
+| 3.16        | 3.27             | 2027            |
++-------------+------------------+-----------------+
+| 3.17        | 3.28             | 2028            |
++-------------+------------------+-----------------+
+| 3.18        | 3.29             | 2029            |
++-------------+------------------+-----------------+
+| 3.19        | 3.30             | 2030            |
++-------------+------------------+-----------------+
+| 3.20        | 3.31             | 2031            |
++-------------+------------------+-----------------+
+| 3.21        | 3.32             | 2032            |
++-------------+------------------+-----------------+
+| 3.22        | 3.33             | 2033            |
++-------------+------------------+-----------------+
+| 3.23        | 3.34             | 2034            |
++-------------+------------------+-----------------+
+| 3.24        | 3.35             | 2035            |
++-------------+------------------+-----------------+
+| 3.25        | 3.36             | 2036            |
++-------------+------------------+-----------------+
+
+
+Security implications
+=====================
+
+None known. No change to durations or timing of bug fix and security phases.
+
+How to teach this
+=================
+
+We will announce this on blogs, in the 3.14 release notes, documentation,
+and through outreach to the community.
+
+This change targets the version following 3.14:
+instead of 3.15 it will be 3.26.
+This PEP was proposed in June 2024.
+Development for the 3.15/3.26 release will begin in May 2025,
+with the first alpha in October 2025 and initial release in October 2026.
+We can already update documentation during the 3.14 cycle.
+This gives plenty of notice.
+
+We can make preview builds which only change the version for early testing.
+
+We could ship a ``python3.15`` command as part of Python 3.26 that immediately
+errors out and tells the user to use ``python3.26`` instead.
+
+.. _PEP 2026 Rejected:
+
+Rejected ideas
+==============
+
+YY.0
+----
```

Comment:
```suggestion
YY.0 versioning
---------------
```

Reply:
I'm going to keep these headers more concise, but I've applied all other other suggestions, thanks!